Gems/ScriptCanvas/Code/Editor/View/Windows/Tools/InterpreterWidget/InterpreterWidget.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <QMessageBox>
+
+#include <AzToolsFramework/API/ToolsApplicationAPI.h>
+#include <UI/PropertyEditor/ReflectedPropertyEditor.hxx>
+
+#include <Editor/View/Windows/Tools/InterpreterWidget/InterpreterWidget.h>
+#include <Editor/View/Windows/Tools/InterpreterWidget/ui_InterpreterWidget.h>
+
+namespace InterpreterWidgetCpp
+{
+    using namespace ScriptCanvasEditor;
+
+    AZStd::pair<bool, bool> ToStartStopButtonEnabled(InterpreterStatus status)
```

Comment:
Could use a comment that explains the meaning of the return values

Reply:
X